sbin/submitty_autograding_shipper.py

```diff
@@ -746,30 +746,31 @@ def shipper_process(my_name,my_data,full_address,which_untrusted,overall_lock):
     """
 
     which_machine   = full_address
-    my_capabilities = my_data[my_name]['capabilities']
+    my_capabilities = my_data['capabilities']
+    my_folder = worker_folder(my_name)
 
     # ignore keyboard interrupts in the shipper processes
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
     counter=0
     while True:
         try:
-            my_job = get_job(my_name,which_machine,my_capabilities,which_untrusted,overall_lock)
+            my_job = get_job(my_name,which_machine,my_capabilities,which_untrusted)
             if not my_job == "":
                 counter=0
-                grade_queue_file(my_name,which_machine,which_untrusted,os.path.join(INTERACTIVE_QUEUE,my_job))
+                grade_queue_file(my_name,which_machine,which_untrusted,os.path.join(my_folder, my_job))
                 continue
             else:
                 if counter == 0 or counter >= 10:
-                    print ("{0} {1}: no available job".format(my_name, which_untrusted))
+                    autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID,
+                        message="{0} {1}: no available job".format(my_name, which_untrusted))
```

Comment:
Are we going to continue to use `string.format`? Or use `f-Strings` for python string formatting moving forward: https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36

Reply:
Yes, we should be moving to ```f-strings```. Our current policy is to replace instances of ```.format()``` as we encounter them. @hdrodz97 , feel free to clean this one up.